README.md

```diff
@@ -19,5 +19,20 @@ Building for electron seems to require the use of electron-rebuild:
 
 # Usage
 
-See example.js for how to use the module
+how to use the module:
```

Comment:
I would stick to document style with capitalization, i.e. "How to use the module:"